src/server/biosecurity-map/upload-plan/index.js

```diff
@@ -39,15 +40,28 @@ export class UploadPlanController extends QuestionPageController {
 
     const data = JSON.parse(response.payload.toString())
 
-    req.yar.set('upload', data)
+    const answer = new this.page.Answer({
```

Comment:
Should this.page not be uploadPlanPage ?

Reply:
effectively the same due to line 74
new UploadPlanController(uploadPlanPage)